Mixmatch/streetview_dataset/parse_TFRecord.ipynb

```diff
@@ -0,0 +1,221 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import tensorflow as tf\n",
+    "import matplotlib.pyplot as plt\n",
+    "import IPython.display as display\n",
+    "import itertools\n",
+    "import os  # used for directory operations\n",
+    "import io\n",
+    "from PIL import Image  # used to read images from directory\n",
+    "\n",
+    "tf.enable_eager_execution()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def read_TFRecord(file_path):\n",
+    "    raw_image_dataset = tf.data.TFRecordDataset(file_path)\n",
+    "\n",
+    "    # Create a dictionary describing the features.\n",
+    "    image_feature_description = {\n",
+    "        'image/source_id': tf.io.FixedLenFeature([], tf.string),\n",
+    "        'image/object/bbox/confidence': tf.io.VarLenFeature(tf.float32),\n",
+    "        'image/object/bbox/xmin': tf.io.VarLenFeature(tf.float32),\n",
+    "        'image/object/bbox/ymin': tf.io.VarLenFeature(tf.float32),\n",
+    "        'image/object/bbox/xmax': tf.io.VarLenFeature(tf.float32),\n",
+    "        'image/object/bbox/ymax': tf.io.VarLenFeature(tf.float32),\n",
+    "    }\n",
+    "\n",
+    "    def _parse_image_function(example_proto):\n",
+    "      # Parse the input tf.Example proto using the dictionary above.\n",
+    "      return tf.io.parse_single_example(example_proto, image_feature_description)\n",
+    "\n",
+    "    parsed_image_dataset = raw_image_dataset.map(_parse_image_function)\n",
+    "    return parsed_image_dataset\n",
+    "\n",
+    "def parse_labels(image_features):\n",
+    "    img_name = str(image_features['image/source_id'].numpy()).split('/')[-1][:-1]\n",
+    "    confidence = tf.sparse_tensor_to_dense(image_features['image/object/bbox/confidence'], default_value=0).numpy()\n",
```

Comment:
Add a library to define those common const strings: 'image/object/bbox/confidence', etc.

Reply:
I've added a global constant for these.